SWLOR.Game.Server/Feature/SpawnDefinition/HutlarSpawnDefinition.cs

```diff
@@ -51,5 +53,27 @@ private void Valley()
                 .WithFrequency(8)
                 .RandomlyWalks();
         }
+        private void Wastes()
+        {
+            _builder.Create("HUTLAR_WASTES", "Hutlar Frozen Wastes")
+                .AddSpawn(ObjectType.Creature, "frost_saber")
+                .WithFrequency(8)
+                .RandomlyWalks()
+
+                .AddSpawn(ObjectType.Creature, "frost_saber_cub")
+                .WithFrequency(6)
+                .RandomlyWalks();
+        }
+        private void Frzcave()
```

Comment:
Same comment here about using the full wording of "FrozenCave" here and the identifier below.